wagtail/contrib/modeladmin/views.py

```diff
@@ -34,6 +33,18 @@
 from .forms import ParentChooserForm
 
 
+try:
```

Comment:
you might have also look into this way of django-filter adding compatibility for query terms https://github.com/carltongibson/django-filter/pull/921/files

Reply:
Do you mean https://github.com/carltongibson/django-filter/pull/851 ?

921 is exactly what is implemented here.  It seems the most 'safe' way of solving this for now; note my comments about this in the PR description.

Reply:
got it